src/templates/items/parts/item-duration.hbs

```diff
@@ -0,0 +1,29 @@
+<div class="form-group input-select">
+    <label>{{localize "SFRPG.EffectDurationTypesLabel"}}</label>
+    <div class="form-fields">
+        <input type="text" name="system.activeDuration.value" value="{{itemData.activeDuration.value}}" data-dtype="Number" placeholder="-"/>
+        <select name="system.activeDuration.unit">
+            {{#select itemData.activeDuration.unit}}
+            <option value=""></option>
+            {{#each config.effectDurationTypes as |name key|}}
+            <option value="{{key}}">{{localize name}}</option>
+            {{/each}}
+            {{/select}}
+        </select>
+    </div>
+</div>
+<div class="form-group input-select">
+    <label>{{localize "SFRPG.EffectEndTypesLabel"}}</label>
+    <div class="form-fields">
+        <input type="text" name="system.sourceActorId" value="{{itemData.sourceActorId}}" data-dtype="String" placeholder="{{actor.id}}"/>
```

Comment:
This could do with some kind of tooltip, as it is quite an advanced feature.
I also think it's worth adding a dropdown to simplify picking the source actor a bit. It could either have two options "Parent actor" or "other (insert ID)", or alternatively auto-populate from combatants in the current combat (defaulting to just "Parent actor" (or maybe a list of all PCs) when there is no combat